### PR TITLE
spec: Populate Introduction

### PR DESCRIPTION
**Pull request by odejesush**
This commit populates the Introduction section of the spec using the Introduction of the explainer as reference.

***
<a href="https://pr-preview.s3.amazonaws.com/odejesush/controlled-frame/pull/15.html" title="Last updated on May 9, 2023, 3:51 PM UTC (ec9cd33)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/chasephillips/controlled-frame/15/3f86fc3...odejesush:ec9cd33.html" title="Last updated on May 9, 2023, 3:51 PM UTC (ec9cd33)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
 
 # Introduction # {#introduction}
 
-Hello world!
\ No newline at end of file
+This specification describes a content embedding API that satisfies some
+critical use cases that <{iframe}> does not support. This embedding environment
```
Comment (chasephillips) [inline index.bs:18]: "use cases for IWAs"

File: index.bs [hunk 2]
```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
...
+should allow embedding all content, including content which <{iframe}> cannot
+embed, and provide embedding sites more controlover that embedded content.
```
Comment (chasephillips) [inline index.bs:20]: nit: "control over"

File: index.bs [hunk 3]
```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
...
+Additionally, embedding a site's content should be possible without express
+permission from the embedded site.
```
Comment (chasephillips) [inline index.bs:22]: Re: "should allow embedding all content" and "should be possible without express permission", I think these 2 phrases refer to the same capability (ignoring X-Frame-Options). Do they? If so, let's just refer to it once here by merging the statements together.

File: index.bs [hunk 4]
```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
...
+
+Since this is a particularly powerful API, it use and availability makes an app
```
Comment (chasephillips) [inline index.bs:24]: s/it use/its use/

File: index.bs [hunk 5]
```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
...
+a target of various types of hacking. As a result, this API is limited to use in
+[Isolated Web Applications](https://github.com/WICG/isolated-web-apps/blob/main/README.md)
+(IWAs) which have addtional safeguards in place to protect users and developers.
+IWAs are not a normal web application and can exist only at a special
+'isolated-app:' scheme. This means that by design, this API will not be
```
Comment (chasephillips) [inline index.bs:29]: let's keep the phrasing: "This means by design that this API will not be"

File: index.bs [hunk 6]
```diff
@@ -14,4 +14,21 @@ Group: WebView-CG
...
+This specification describes a content embedding API that satisfies some
+critical use cases that <{iframe}> does not support. This embedding environment
+should allow embedding all content, including content which <{iframe}> cannot
+embed, and provide embedding sites more controlover that embedded content.
+Additionally, embedding a site's content should be possible without express
+permission from the embedded site.
+
+Since this is a particularly powerful API, it use and availability makes an app
+a target of various types of hacking. As a result, this API is limited to use in
+[Isolated Web Applications](https://github.com/WICG/isolated-web-apps/blob/main/README.md)
```
Comment (chasephillips) [inline index.bs:26]: In the preview, this isn't correctly linked. Perhaps this needs to be a bikeshed manual biblio entry and linked from here.

**Review by chasephillips — CHANGES_REQUESTED** [review]

File: index.bs (hunk 6 above)
Comment (odejesush) [reply index.bs]: Thanks, I completely missed that this wasn't linked properly. Looks like Bikeshed supports some Markdown, but not this linking syntax.

File: index.bs (hunk 3 above)
Comment (odejesush) [reply index.bs]: Yes, they do go over the same capability. I merged these two as you suggested. Thanks!

File: index.bs (hunk 6 above)
Comment (odejesush) [reply index.bs]: I did figure out how to add this as a biblio entry. Let me know which approach you prefer.

**Review by chasephillips — APPROVED** [review]
LGTM, feel free to land either way now (either pre-squashed or post-PR-squashed)

File: index.bs [hunk 7]
```diff
@@ -12,6 +12,18 @@ Abstract: TODO
 Group: WebView-CG
 </pre>
 
+<pre class="biblio">
```
[the remark this answers is not in the document]
Comment (odejesush) [reply index.bs:18]: Sounds good, thanks!

File: index.bs (hunk 6 above)
Comment (chasephillips) [reply index.bs]: Biblio looks great!

**Review by chasephillips — APPROVED** [review]
LGTM!

**Pull request merged**